src/utils/browserUtil.ts

```diff
@@ -0,0 +1,9 @@
+import * as openpackage from 'open';
```

Comment:
What do you think about using _open as the name here? openpackage seems a bit odd to me 😊

Reply:
Makes sense 😄 Naming is hard...